setup.py

```diff
@@ -388,7 +388,7 @@ def make_relative_rpath(path):
     extras_require={
         # On Cloud TPU VM install with:
         # $ sudo pip3 install torch_xla[tpuvm] -f https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.11-cp38-cp38-linux_x86_64.whl
-        'tpuvm': [f'libtpu-nightly @ {_libtpu_storage_path}'],
+        'tpuvm': [f'libtpu-nightly @ {_libtpu_storage_path}', "expecttest"],
```

Comment:
I don't think this is the right place, we should install it in docker setup directly(I believe our CI docker does the same). For the user of our wheel, they don't need `expecttest`. This is only for our unit test.

Reply:
We should add a new `extras_require` called e.g. `test` that bundles together common test dependencies. Then test environments only need to run `pip install torch_xla[test]`, and non-test users don't get unnecessary dependencies

Reply:
so should we add "expecttest" in these file pathes both?:
(1)`.circleci/docker/Dockerfile` for CI docker
(2)`setup.py` for user

Reply:
`.circleci/docker/Dockerfile` is not the docker used for TPUVM I think. I think it is https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile. @steventk-g can confirm. You can also confirm by manually trigger TPU CI in this pr.

Reply:
Correct. It's the experimental Dockerfile

Reply:
> `.circleci/docker/Dockerfile` is not the docker used for TPUVM I think. I think it is https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile. @steventk-g can confirm. You can also confirm by manually trigger TPU CI in this pr.

Thanks, checked trigger TPU CI's result in https://github.com/pytorch/xla/pull/4495, and updated to `[docker/experimental/Dockerfile](https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile)`

Reply:
> updated to `[docker/experimental/Dockerfile](https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile)`

Thanks, updated in updated to [docker/experimental/Dockerfile](https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile)